### ISSUE TITLE
0010796:
Contracts not loading

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#5344]
**Reported by schatzs on 2 Feb 2015 10:13**

**Version:** Koriander (2014.09.7)

The contracts view in the sales module does not load entries.

Firebug reports:
Zeitstempel: 02.02.2015 11:13:12
Fehler: uncaught exception: No keyfield config found for Abrechnungsperiode in Sales registry.

**Comment by Gloirin** [comment]
**Comment posted by mspahn on 3 Feb 2015 16:30**

Have you skipped any updates?

Does php setup.php --update help you?

**Comment by Gloirin** [comment]
**Comment posted by schatzs on 4 Feb 2015 09:40**

No, I installed all updates up to 2014.09.7. Yesterday I also tested the manual update to the latest release 2014.09.7 concerning http://wiki.tine20.org/Update_Howto to foreclose possible problems with the cli update
Result was the same. Before the last update the everything works fine.
Could a reason be that the site is running through https or was something changed that I have to consider in the apache/htaccess settings?
